### PR TITLE
Pixhawk 6x pro

### PR DESCRIPTION
**Pull request by YogeshMaan**
added holybro_pixhawk6x_pro and updated holybro_pixhawk6x

### CODE REVIEW

**Comment by vincentpoont2** [comment]
@hamishwillee Can we add this into the PX4 Guide?

**Comment by hamishwillee** [comment]
I will review tomorrow.

File: en/flight_controller/pixhawk6x_pro.md [hunk 1]
```diff
@@ -5,15 +5,20 @@ PX4 does not manufacture this (or any) autopilot.
...
+- All new advanced durable vibration isolation material with resonance frequency in the higher spectrum, ideal for industrial and commercial drone applications
 - High performance STM32H753 Processor
 - Ethernet interface for high-speed mission computer integration
 
+## Baseboards
+
+The Pixhawk 6X Pro can be purchased with a number of baseboards (or no baseboard) to suit different use cases and vehicle types, including [Standard v2A](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-baseboard-v2-ports), [Standard v2B](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-baseboard-v2-ports), and [Mini](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-mini-baseboard-ports), which are shown below.
+It can also be used with any other Pixhawk Autopilot Bus (PAB) specification-compliant baseboard, such as the [Holybro Pixhawk Jetson Baseboard](../companion_computer/holybro_pixhawk_jetson_baseboard.md) and [Holybro Pixhawk RPI CM4 Baseboard](../companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md).
+
```
Comment (hamishwillee) [inline en/flight_controller/pixhawk6x_pro.md:21]: FYI I added this because the baseboards felt like they were just floating around with no explanation.

**Review by hamishwillee — APPROVED** [review]
Really great job thanks @YogeshMaan !!!

I didn't do very much to this other than running prettier and adding to the autopilot pixhawk standard page.

**Pull request merged**